src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaverForBarn.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import java.util.UUID
+
+data class OppgaveForBarn(val id: UUID,
+                          val beskrivelse: String)
```

Comment:
For meg virker det som att `OppgaveForBarn` i ef-sak inneholder noen flere felt, burde de fjernes, eller legges til her?

Reply:
Tar med kun de feltene i sak også (hvis ikke vi finner ut at vi skal sende med alle felter derifra, da)